Week1/hm/queries5.js

```diff
@@ -0,0 +1,39 @@
+const mysql = require('mysql');
+const connection = mysql.createConnection({
+    host: 'localhost',
+    user: 'hyfuser',
+    password : 'hyfpassword',
+    database: 'world'
+});
+
+connection.connect();
+
+const queries =[
+    // cities with population in between 500,000 and 1 million
```

Comment:
Careful here: the comments don't match what the queries do :)

Reply:
Thank you a lot for review and feedback . 
I’ll check again. :)